src/ctl.c

```diff
@@ -382,11 +382,10 @@ static const ctl_named_node_t opt_node[] = {
 	{NAME("prof_final"),	CTL(opt_prof_final)},
 	{NAME("prof_leak"),	CTL(opt_prof_leak)},
 	{NAME("prof_accum"),	CTL(opt_prof_accum)},
-	{NAME("prof_recent_alloc_max"), CTL(opt_prof_recent_alloc_max)},
-	{NAME("prof_experimental_use_sys_thread_name"),
-	    CTL(opt_prof_experimental_use_sys_thread_name)},
-	{NAME("zero_realloc"),	CTL(opt_zero_realloc)},
-	{NAME("prof_time_resolution"),	CTL(opt_prof_time_res)}
+	{NAME("prof_recent_alloc_max"),	CTL(opt_prof_recent_alloc_max)},
+	{NAME("prof_sys_thread_name"),	CTL(opt_prof_sys_thread_name)},
```

Comment:
Any reason to drop `experimental` here?

I don't think it *really* matters, since we're not documenting this for a while and have warned all known users about possible semantics changes, but it doesn't hurt either.

Reply:
No particular reason, other than that "experimental" is a pretty long word.

Also, like you said, anything we're not documenting is kind of automatically "experimental". I don't think this system thread name feature is strikingly "more experimental" than other "experimental" features e.g. the high resolution timer, so we don't really have to stress on "experimental" for it.